test/extended/baremetal/hosts.go

```diff
@@ -187,3 +187,81 @@ var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] Baremetal platfor
 		o.Expect(found).To(o.BeFalse())
 	})
 })
+
+var _ = g.Describe("[sig-installer][Feature:baremetal][Serial] A baremetal deployment without a provisioning network should", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc = exutil.NewCLI("baremetal")
+	)
+
+	g.It("show the Provisioning Network as 'Disabled'", func() {
+		skipIfNotBaremetal(oc)
+
+		dc := oc.AdminDynamicClient()
+
+		skipIfProvisioningNetworkSet(dc)
+
+		o.Expect(getProvisioningNetwork(dc)).To((o.BeEquivalentTo("Disabled")))
+
+		g.By("Not allow setting the ProvisioningNetwork to 'Managed' with invalid values")
```

Comment:
I set this as a continuation of the same case as it uses the same parameters
I could separate it, but it would just duplicate the authentication part and leave only L205 to the precedent test